src/app/manuals/herdManual.ts

```diff
@@ -8,50 +8,50 @@ export function herdDemo(): void {
     {
       name: AnimalNames.RABBIT,
       tradeValue: 1,
-      path: '/static/images/avatars/rabbit.png',
+      path: './static/images/avatars/rabbit.svg',
```

Comment:
IMO, it will be good for such changes to be done in a separate PR with e.g. 'fix' label.